docs/products/cassandra/howto/disable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,113 @@
+
+About disabling CCR
+-------------------
+
+When you enable CCR for your service, you connect it to another service, which results in creating a CCR pair of services. You can disable CCR for your service either by splitting the services constituting the CCR pair or by deleting one of them.
+
+* `Aiven console <https://console.aiven.io/>`_
+* CLI
+* API
+
+It's recommended to use the Aiven console for disabling CCR.
+
+Prerequisites
+-------------
+
+* Aiven account
+* Depending on the method you choose to use for enabling CCR
+
+  * Access to the `Aiven console <https://console.aiven.io/>`_
+  * `cURL` CLI tool
+  * `Aiven CLI tool <https://github.com/aiven/aiven-client>`_
+
+* CCR enabled on a pair of Aiven for Apache Cassandra services
+
+Disable CCR in the console
+--------------------------
+
+In the console, use either of the two following methods to disable CCR on your service(s): split the services or delete one of them.
+
+Split the services
+''''''''''''''''''
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select the service for which you'd like to disable CCR.
+3. In the **Overview** tab of the service's page, navigate to **Cross Cluster Replication** and select **Split cluster**.
+
+   .. warning::
+
+      As soon as you split the cluster, the two services constituting the CCR pair become independent. It's not possible to recreate the CCR pair connecting back to the same service. To enable CCR on your service again, you can create a new service and CCR-connect it to your existing service.
+   
+4. When in the **Warning** popup, make sure you understand the impact and select **Split cluster**.
+
+.. topic:: Result
+
+   You service no longer replicates to the other service since the services have been disconnected.
+
+Delete a service
+''''''''''''''''
+
+To disable CCR on your service, delete the service that is connected to your services for CCR purposes.
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select an Aiven for Apache Cassandra service on which you'd like to enable CCR.
+3. In the **Overview** tab of the service's page, navigate to the **Cross Cluster Replication** section and select the name of the service provided in the CCR description, which is supposed to take you to the service's page.
+4. In the **Overview** tab of the service, select **Delete service** from the top right corner.
+
+   .. warning::
+
+      As soon as you delete the service where your data has been replicated, CCR gets disabled and your data is no longer replicated between regions.
+
+5. When in the **Delete confirmation** popup, make sure you understand the impact, copy-paste the service name, and select **Delete**.
+
+.. topic:: Result
+
+   You've disabled CCR on your service by deleting the service used as a sink for replicated data.
```

Comment:
> used as a sink for replicated data

hints that the other service is a source, meaning that the other service is somehow different; maybe "by deleting one of the peer services in a CCR service pair"?